server/controllers/profile.js

```diff
@@ -114,4 +116,39 @@ exports.findAllProfiles = asyncHandler(async (req, res, next) => {
       }
 
     res.status(200).json({ profiles: profiles });
```

Comment:
This should be one profile, `profile`. You don't have `profiles` defined!

Reply:
I don't know exactly how this function works, cause I didn't create it.